cishouseholds/pipeline/high_level_transformations.py

```diff
@@ -1457,37 +1458,37 @@ def derive_work_status_columns(df: DataFrame) -> DataFrame:
     df = update_column_values_from_map(df=df, column="work_status_v2", map=work_status_dict["work_status_v2"])
 
     ## Not needed in release 1. Confirm that these are v2-only when pulling them back in, as they should likely be union dependent.
```

Comment:
Do these need to apply to digital as well? Currently only called inside the transform_survey_responses_version_2_delta function

Reply:
Solved by moving to union_dependent_derivations to apply to v2 and digital